src/components/runbutton.js

```diff
@@ -1,38 +1,31 @@
 import React from 'react';
+import Button from '@material-ui/core/Button';
 
-const RunButton = ({ taskData, gridCapaJobLauncherUrl }) => {
+const RunButton = ({ taskData }) => {
     let taskStatus = taskData === null ? 'Not created' : taskData.status;
     let taskTimestamp = taskData === null ? 'Not created' : taskData.timestamp;
 
-    function launchTask() {
-        const request = new Request(gridCapaJobLauncherUrl + 'jobs/', {
+    const launchTask = () => fetchRunButtonPost();
+
+    function fetchRunButtonPost() {
+        console.log("Call run for task:" + taskTimestamp);
+        const requestOptions = {
             method: 'POST',
-            body: JSON.stringify({
-                id: taskTimestamp,
-            }),
-        });
-        fetch(request)
-            .then((response) => {
-                if (response.status === 200) {
-                    return response.json();
-                } else {
-                    console.log(response.status);
-                    throw new Error('Something went wrong on api server!');
-                }
-            })
-            .then((response) => {
-                console.debug(response);
-                // ...
-            })
-            .catch((error) => {
-                console.error(error);
-            });
+            headers: { 'Content-Type': 'application/json' }
+        };
+        fetch('http://localhost/core/valid/job/start/' + taskTimestamp, requestOptions)
+            .then(response => response.json())
+            .then();
     }
 
     return (
-        <button disabled={taskStatus !== 'READY'} onClick={launchTask}>
+        <Button color="primary"
```

Comment:
it would be good to also add an inputProps property with a 'data-test' field to be able to handle this component in acceptance tests